src/pybind/mgr/orchestrator/module.py

```diff
@@ -592,8 +592,8 @@ def _mds_add(self, fs_name, placement=None):
     def _rgw_add(self, realm_name, zone_name, placement=None, inbuf=None):
         usage = """
 Usage:
-  ceph orch daemon rgw add -i <json_file>
-  ceph orch daemon rgw add <realm_name> <zone_name>
+  ceph orch daemon add rgw -i <json_file>
+  ceph orch daemon add rgw <realm_name> <zone_name>
```

Comment:
ops this might have snuck through the rebase.. will fix this.

Reply:
actually this is corrected, but I guess out of scope for this change. But can fix it when I get the RGW SSL patch landed :)